airbyte-integrations/connectors/source-jotform/metadata.yaml

```diff
@@ -0,0 +1,35 @@
+metadataSpecVersion: "1.0"
+data:
+  allowedHosts:
+    hosts:
+      - "*"
```

Comment:
Please add the api host that's actually being used here.

Reply:
Fixed @natikgadzhi